client/extensions/wp-super-cache/controller.js

```diff
@@ -17,8 +17,8 @@ import WPSuperCache from './main';
 
 const controller = {
 
-	settings: function( context ) {
-		const siteId = getSiteFragment( context.path );
+	settings: function(context, next) {
+	    const siteId = getSiteFragment( context.path );
```

Comment:
Yes, definitely something I've encountered... Unfortunately, I don't have a good fix for this yet.